### DISCUSSION TITLE
Correct alpha sorting of albums with numbers in the title

### DISCUSSION THREAD

**Discussion by Apesbrain** [LMS-Community/slimserver#886] [Ideas]
Is there a change that could be made to LMS to correctly alphabetically sort albums such as below:

![Clipboard01](https://user-images.githubusercontent.com/26585705/235303430-c346b345-7bba-49f8-96d4-008c5e5aff20.jpg)

I know it could be done with an ALBUMSORT tag, but that's a tedious job if you have already number of such albums.  Thanks.

**Comment by michaelherger** [comment]
I'd say you'll have to bite the bullet. You'd always have to make assumptions when trying to automate things. But what would the assumption be here? Pad all numbers in titles with leading zeros so they'd (probably) sort in the correct order? But what about the years and dates? You posted a great example, as even some of the dates would be rather difficult to parse.

**Reply by Apesbrain** [reply]
Thanks, you make a good point. When I looked into my library I found it was only the Dead albums that had this issue. Wasn't as tedious as I thought to go in with Mp3tag and create and edit ALBUMSORT tags for these. Seems to have worked well.
